### ISSUE TITLE
Форма зворотнього зв'язку

### ISSUE THREAD

**Issue by chief-j** [hurtom/toloka#83]
https://toloka.to/t9030-1800#896257
Не впевнений, для чього вона стане корисною, можливо для знайдених помилок/багів/побажань

**Comment by chief-j** [comment]
https://toloka.to/t9030-1800#896213

**Comment by yukoff** [comment]
Я створив [Зворотній зв'язок Гуртом](https://quire.io/w/hurtom-feedbacks) саме для цієї мети, тож думаю можна оновити тред на толоці, додати в опис цього issue та закривати.

**Comment by karmeljuk** [comment]
я рахую таку форму корисною, зворотній зв'язок це дуже важливо

**Comment by chief-j** [comment]
@yukoff на сторонніх резурсах ніхто не реєструватиметь тільки, щоб повідомити про баг. Форму треба інтегрувати в форум
Щось подібне до цього https://usabilla.com/graphics/feature/slider/feedback-form.png
Можливо, список стане занадто великим і нічого корисного не знайдеться

**Comment by chief-j** [comment]
Подумав і зрозумів, що така форма не принеле великої користі
Треба зробити спеціальну тему, щоб користувачі могли голлсувати/ставити лайки повідомленню і формувався список популярних коментарів

**Comment by yukoff** [comment]
@chief-j У Quire, на відміну від GitHub, є вхід/реєстрація за допомогою обліковки Ґуґла, тому для нетехнічних людей це зручніше (не треба реєструватись на GitHub, щоб повідомити про проблему). Хоча щодо саме фідбеку, то є сервіси саме під це спроектовані (але здебільшого платні)

Те, що на скріншоті - це скоріше відгук, аніж зворотній зв'язок, там немає:
* можливості відстежувати своє питання, тобто невідомий його статус
* ніяких сповіщень на пошту (в браузер через додаток) про відповіді/коментарі

Проте з подібного відгуку можна зрозуміти, який був настрій в людини та як вона оцінює сайт/сервіс (на скріншоті мова про швидкість сайту).

> Треба зробити спеціальну тему, щоб користувачі могли голлсувати/ставити лайки повідомленню і формувався список популярних коментарів

Зробити спеціальну тему - де? Форуми зазвичай не призначені для такого, до того ж якщо (як в одному з тредів щодо оновлення) буде овер 60 сторінок - це капець (я в тому сенсі, якщо їх розгрібати/аналізувати/тощо).

Вподобання... Я до речі не звернув на це увагу, але здається в новому TP цього все ж немає - теоретично можна зробити, але чи варте воно того? Щоб зробити щось, чим більш-менш можна користуватись (в рамках поставленої задачі) - треба багацько понаписувати (це час), перевірити, отримати гнівних відгуків 😉, модифікувати базу тощо. До того ж, чим важчі будуть запити (бо більше даних тре вибирати з бази, бо ж статистика додасться), тим більше це з рештою буде відбиватись і на роботі анонсера (я маю на увазі, коли велика кількість людей в онлайні - збільшення аудиторії, гадаю, задача не останнього порядку). А Quire (чи щось саме на фідбек орієнтоване, типу uservoice та інші) - бери і користуйся вже зараз.

Щодо простої форми - теж не проблема додати (той же [Typeform](https://www.typeform.com) крутий і легко інтегрується), але тут все те ж саме, що я описував для сервісу зі скріншоту.

**Comment by chief-j** [comment]
> Зробити спеціальну тему - де? Форуми зазвичай не призначені для такого, до того ж якщо (як в одному з тредів щодо оновлення) буде овер 60 сторінок - це капець (я в тому сенсі, якщо їх розгрібати/аналізувати/тощо).

Щоб не читати всі повідомлення, достатньо сортувати їх з найбільшою кількістю підтримки

**Comment by karmeljuk** [comment]
Підтримую ідею Typeform, дуже зручна річ

**Comment by yukoff** [comment]
Так, він зручний (та локалізований), але для користувача це абсолютно не інформативно - ну написав я відгук, а далі, власне, що? ;) Як про стан речей дізнатись (особливо, якщо то було побажання)? Не дарма ж існують всі ці Uservoice-и, GetSatisfaction-и та [інші](https://alternativeto.net/software/uservoice/) ;) Хоча у quire немає голосування як такого, проте там є "вподобайки", відповідно питання, що матимуть найбільше вподобань отримують вищий пріоритет.

Чому ще я зі скепсисом ставлюсь саме до моделі з формою? На додачу до вже сказаного - людина, що писатиме відгук, не може бачити, чи задавались були вже подібні питання. Як результат -
 однотипних питань можуть наплодити овер 9000, ну а потім що? Знов йти на форум та писати в дусі "аааа, адміни нічого не роблять, вони нас не чують, їм плювати на нас" ;)

P.S. Самі Quire (команда) навіть свій [гайд](https://quire.io/w/Getting_Started_with_Quire) там зробили - і скажу, що це дійсно цікава ідея, якщо чогось не допетрав одразу можна спитати в коментарях.

P.P.S. Зовсім забув - не так давно для одного з проектів підбирали подібне рішення, вони наче на [UserReport](https://userreport.com) зупинились, в безкоштовному пакеті було все, що їм було потрібно.
